python-toolbox/setup.py

```diff
@@ -77,7 +77,8 @@
     'joblib>=0.11',
     'autopep8>=1.3.3',
     'progressbar2>=3.34.3',
-    'urllib3>=1.21.1',
+    'urllib3==1.21.1',
+    'unidecode==1.0.23',
     'idna>=2.5',
     'bleach>=1.5.0',
     'numpy==1.13.3'
```

Comment:
Please check issue [MARVIN-38] and commit: f3e318690af1e380e7cfe5564797ba634129fa54
by Taka, for the current build error.

Reply:
Thanks, I just merged with develop into my branch and got this commit